lnrpc/routerrpc/router.proto

```diff
@@ -467,6 +467,66 @@ message SetMissionControlConfigResponse {
 }
 
 message MissionControlConfig {
+    /*
+    The maximum number of payment results that mission control will store.
+    */
+    uint32 maximum_payment_results = 4;
+
+    /*
+    The minimum time that must have passed since the previously recorded failure
+    before we raise the failure amount.
+    */
+    uint64 minimum_failure_relax_interval = 5;
+
+    enum ProbabilityModel {
+        UNKNOWN = 0;
+        APRIORI = 1;
+        BIMODAL = 2;
+    }
+
+    /*
+    ProbabilityModel defines which probability estimator should be used in
+    pathfinding.
+    */
+    ProbabilityModel Model = 6;
+
+    /*
+    EstimatorConfig is populated dependent on the estimator type.
+    */
+    oneof EstimatorConfig {
+        AprioriParameters apriori = 7;
+        BimodalParameters bimodal = 8;
+    }
+}
```

Comment:
could we reorder this so the fields go from `1...n`? Also, any reason to skip n=3?